src/Resources/NotificationDestination.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace OhDear\PhpSdk\Resources;
+
+class NotificationDestination extends ApiResource
+{
+    public int $id;
+
+    public ?int $teamId = null;
```

Comment:
I've made the `$teamId` default to `null`, as it's not included on site-specific responses, so any calls to it would return an error for accessing a property before it's defined.